src/main/java/org/mockito/Mockito.java

```diff
@@ -95,15 +95,18 @@
  * <h3 id="0.1">0.1. <a class="meaningful_link" href="#mockito-android">Mockito Android support</a></h3>
  *
  * With Mockito version 2.6.1 we ship "native" Android support. To enable Android support, add the `mockito-android` library as dependency
- * to your project. This artifact is published to the same mockito organization:
+ * to your project. This artifact is published to the same Mockito organization and can be imported for Android as follows:
  *
  * <pre class="code"><code>
  * repositories { jcenter() }
- * dependencies { testCompile "org.mockito:mockito-android:2.+" }
+ * dependencies { androidTestCompile "org.mockito:mockito-android:2.+" }
```

Comment:
Changed the doc. I think it is best if we do not refer to a version alltogether to keep the documentation "time-less" in case there are old versions somewhere in the future. Most people do know how to specify a version.